commands/deploy.go

```diff
@@ -254,7 +254,7 @@ func deployImage(
 
 	functionResourceRequest1 := proxy.FunctionResourceRequest{}
 	proxy.DeployFunction(fprocess, gateway, functionName,
-		registryAuth, image, language,
+		image, registryAuth, language,
```

Comment:
You're right - I didn't expect these two variables to be changed around.. so we should test with a private repo.